Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -501,6 +504,12 @@ def self.pr_pull
           upload_args << "--root-url-using=#{args.root_url_using}" if args.root_url_using
           safe_system HOMEBREW_BREW_FILE, *upload_args
         end
+      ensure
+        if args.retain_bottle_dir?
+          puts "::set-output name=bottle_path::#{dir}"
```

Comment:
`::set-output` is deprecated.

```suggestion
          File.open(ENV.fetch("GITHUB_OUTPUT"), "a") do |f|
            f.puts "name=bottle_path::#{dir}"
          end
```

Reply:
Thank you!